conda/lib/dependabot/conda/package_manager.rb

```diff
@@ -16,7 +16,7 @@ class CondaPackageManager < Dependabot::Ecosystem::VersionManager
       extend T::Sig
 
       NAME = "conda"
-      VERSION = "latest"
+      VERSION = "23.9.0"  # Use a valid semantic version instead of "latest"
```

Comment:
```suggestion
```

In the constructor we should set the version to `nil` instead of `Dependabot::Conda::Version.new(VERSION)`.